### PR TITLE
Assistant/sara_fe/refactor

### PR DESCRIPTION
**Pull request by hfuturo**

### CODE REVIEW

File: x-men/frontend/public/template/menu.html [hunk 1]
```diff
@@ -7,17 +7,34 @@
...
-        </div>
+    <img src="{{imageUri}}sara.png" alt="SARA logo">
+    <div class="wrapper">
+        <div class="options-wrapper">
+            <div class="option-wrapper">
+                <div class="info-label-wrapper">
+                    <div class="tooltip"> &#9432;
+                        <span class="tooltiptext">
+                            <p>Selects which repository will be analyzed.</p>
+                            <ul>
```
Comment (MRita443) [inline x-men/frontend/public/template/menu.html:18]: <p> element is not permitted as content under <span>html-validate(element-permitted-content)
<ul> element is not permitted as content under <span>html-validate(element-permitted-content)

Maybe change span to div?

File: x-men/frontend/public/template/menu.html [hunk 2]
```diff
@@ -7,17 +7,34 @@
...
+                                    <p><b>Remote</b> will analyze the remote (GitHub) repository.</p>
+                                </li>
+                            </ul>
+                        </span>
+                    </div>
+                    <label for="location">Location:</label>
+                </div>
+                <div>
+                    <select name="location" id="location">
+                        <option value="local" selected="selected">Local</option>
```
Comment (MRita443) [inline x-men/frontend/public/template/menu.html:32]: Attribute "selected" should omit valuehtml-validate(attribute-boolean-style)
```suggestion
                        <option value="local" selected>Local</option>
```

File: x-men/frontend/public/template/menu.html [hunk 3]
```diff
@@ -7,17 +7,34 @@
...
+                        <option value="remote">Remote</option>
+                    </select>
+                </div>
+            </div>
+        <button id="submit">Submit</button>
     </div>
```
Comment (MRita443) [inline x-men/frontend/public/template/menu.html:38]: You're missing a closing tag for a <div>

File: x-men/frontend/public/template/menu.html [hunk 4]
```diff
@@ -7,17 +7,34 @@
...
+                    <label for="location">Location:</label>
+                </div>
+                <div>
+                    <select name="location" id="location">
+                        <option value="local" selected="selected">Local</option>
+                        <option value="remote">Remote</option>
+                    </select>
+                </div>
+            </div>
+        <button id="submit">Submit</button>
```
Comment (MRita443) [inline x-men/frontend/public/template/menu.html:37]: <button> is missing recommended "type" attributehtml-validate(no-implicit-button-type)
```suggestion
        <button id="submit" type="submit">Submit</button>
```

**Review by MRita443 — CHANGES_REQUESTED** [review]
Looks great, just some HTML guidelines my IDE nagged me about

**Review by MRita443 — APPROVED** [review]
LGTM :)

**Review by IgorAndrade10 — APPROVED** [review]

**Review by IgorAndrade10 — COMMENTED** [review]
Looks good

**Pull request merged**